src/betterproto/__init__.py

```diff
@@ -430,12 +429,6 @@ def parse_fields(value: bytes) -> Generator[ParsedField, None, None]:
 
 
 class ProtoClassMetadata:
-    oneof_group_by_field: Dict[str, str]
```

Comment:
IMO it saves on duplication and looks nicer.

Reply:
Having it on the class serves as a single reference point for documenting the class rather than having the types mixed in the the logic. I'm not sure why you'd need to duplicate it anywhere.

Reply:
Well mixing it in with the logic can again in my opinion make it clearer what the attribute is

Reply:
I think it's more important to document the class, since that's conventionally how types work, being able to get the full picture of a class' content at a glance without scanning through the logic is important. Honestly I find the logic ends up harder to read jammed full of types annotations, whereas referencing the class as needed doesn't add much burden to reading.